src/api/schemas/user-schema.ts

```diff
@@ -92,5 +96,17 @@ const updateUserSchema = object({
     })
 });
 
+const addNewUserActivity = object({
+    params: object({
+        id: firebase_id
+    }).strict('Request URL contains an invalid key'),
+    body: object({
+        // Activity
+        activity: activitySchema.strict('Activity field contains an invalid key')
+        // Address
+        // address: addressSchema.strict('Address field contains an invalid key')
```

Comment:
remove commented elements

Reply:
Also, the commented `// Activity` and `// Address` is a little bit redundant as you have the key to tell you what is really happening.

Reply:
Yessir